pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -182,6 +182,11 @@ def test_date_time():
     fname = os.path.join(dirpath, "datetime.csv")
     df0 = pd.read_csv(fname, parse_dates=['Date1', 'Date2', 'DateTime',
                                           'DateTimeHi', 'Taiw'])
+    # GH 19732: Timestamps imported from sas will incur floating point errors
```

Comment:
why did this need to be changed?

Reply:
I believe there are floating point errors when importing dates from the sas file.

```
In [5]: pd.read_sas('pandas/tests/io/sas/data/datetime.sas7bdat')['DateTimeHi']
Out[5]:
0   1677-09-21 00:12:43.145225525
1   1960-01-01 00:00:00.000000000
2   2016-02-29 23:59:59.123456001
3   2262-04-11 23:47:16.854774475
Name: DateTimeHi, dtype: datetime64[ns]

# This matches dates in datetime.csv
In [6]: pd.read_csv('pandas/tests/io/sas/data/datetime.csv')['DateTimeHi']
Out[6]:
0    1677-09-21 00:12:43.145226
1    1960-01-01 00:00:00.000000
2    2016-02-29 23:59:59.123456
3    2262-04-11 23:47:16.854774
Name: DateTimeHi, dtype: object
```

I am not familiar with read_sas or the sas file that was created, but I am fairly certain it's due to the floating point errors

Reply:
can you round these instead of specifying exact values